try-greg.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import math
+
+def make_blanket( blanket_radius, blanket_width, blanket_height, shell_thickness ):        
+    cubit.cmd( f"create brick x {blanket_height} y {blanket_width} z {blanket_width}" )
+    shell_volume_id = cubit.get_last_id( "volume" )
+    cubit.cmd( f"modify curve {get_closest_curve_in_volume_to_location( shell_volume_id, (blanket_height/2, blanket_width/2, 0) )} blend radius {blanket_radius}" )
+    cubit.cmd( f"modify curve {get_closest_curve_in_volume_to_location( shell_volume_id, (-blanket_height/2, blanket_width/2, 0) )} blend radius {blanket_radius}" )
+
+    cubit.cmd( f"create sheet offset from surface in vol {shell_volume_id} offset {-1*shell_thickness} " )
+    inner_volume_id = cubit.get_last_id( "volume" )
+
+    before_vol_ids = cubit.get_entities( "volume" )
+    cubit.cmd( f"subtract volume {inner_volume_id} from volume {shell_volume_id} keep_tool" )
+    after_vol_ids = cubit.get_entities( "volume" )
+    inner_volume_id, shell_volume_id = get_operation_return_ids( before_vol_ids, after_vol_ids )
```

Comment:
When doing Booleans it is possible to generate multiple volumes.  For example:

```
reset
bri x 0.1 y 1 z 1
bri x 0.1 y 0.1 z 1
subtract vol 2 from vol 1
```

![image](https://github.com/makeclean/why/assets/868479/29b55af8-a617-44c3-8fb8-2c4c340bfd70)

And sometimes a volume ID may not change -- for example the tool id if `keep_tool` is used, or with unites:

```
reset
bri x 1
sphere r .5
move vol 2 x .5 y .5 z .5
unite vol 1 2
```
![image](https://github.com/makeclean/why/assets/868479/d491e4e2-00a5-4390-8f12-c7539b3d6b2c)

In the latter case we can see that while the "new" volume is `volume 1` the result of `cubit.get_last_id( "volume" )` is `2` -- the purpose of that function is to provide insight as to what id _the next created_ volume will be assigned (i.e.,  `next_vol_id = cubit.get_last_id( "volume" ) + 1` )

So I've written a little helper-function, `get_operation_return_ids()` that will find ids that remain the same as well as newly constructed entity ids.